### PR TITLE
Bump two packages for the python 3.9 switch

### PR DESCRIPTION
**Pull request by brianmwaters**
Gentoo recently switched to using python 3.9 as the default, and these two packages were left behind on my system. I am sure there are others that need to be bumped, but this is what affected me.

This makes changes to `*-9999.ebuild` packages, which are symlinked to by actual versioned packages. I don't know what the policy is there but those packages may need to be incremented to have an `-r1` suffix.

### CODE REVIEW

**Comment by blshkv** [comment]
Please sign your commits.

File: app-exploits/ropper/ropper-9999.ebuild [hunk 1]
```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DISTUTILS_USE_SETUPTOOLS=rdepend
-PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_COMPAT=( python3_{6..9} )
```
Comment (blshkv) [inline app-exploits/ropper/ropper-9999.ebuild:7]: Please change to 7..9 to be inline

**Review by blshkv — CHANGES_REQUESTED** [review]

File: app-exploits/ropper/ropper-9999.ebuild (hunk 1 above)
Comment (brianmwaters) [reply app-exploits/ropper/ropper-9999.ebuild]: Done, thanks

Comment (blshkv) [reply app-exploits/ropper/ropper-9999.ebuild]: no, 7..9 please.

Comment (blshkv) [reply app-exploits/ropper/ropper-9999.ebuild]: no, 7..9 please.

**Review by blshkv — APPROVED** [review]

**Comment by blshkv** [comment]
I'm still not able to merge, "The base branch requires all commits to be signed."

**Comment by brianmwaters** [comment]
I don't know what I'm doing, so I just force-pushed.

Github is saying the commits are "unverified" because my key is expired, but it doesn't expire until June 20, so I'm not sure what's wrong.

Let me know if they need to be signed by a newer key and I'll jump through the hoops.

**Comment by brianmwaters** [comment]
Nevermind, it's the old (pre-force push) commits that it's saying are "unverified". I don't know why it says the key is expired because they aren't signed at all. Anyway I think this is all good now.

**Comment by blshkv** [comment]
done, thanks!

**Pull request merged**